jts-transaction-service/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM fedora:22
+MAINTAINER Gytis Trikleris <gytis@redhat.com>
+WORKDIR /home
+
+ENV NARAYANA_PORT 4710
+ENV NARAYANA_VERSION 5.2.1.Final
+ENV NARAYANA_HOME /home/narayana
+ENV JACORB_LOG_LEVEL 3
```

Comment:
Weird stuff, I just wanted to ask you about the version: `docker info` and `docker version`.